### ISSUE TITLE
How to calculate the scale_factor for sketch-rnn dataset?

### ISSUE THREAD

**Issue by shanjgit** [tensorflow/magenta-js#387]
As the document says, 

> "all the offset data has been normalized to have a standard deviation of 1.0 on the training set, after simplifying the strokes. ...Thus each dataclass has its own scale_factors to scale down, and these numbers are usually between 60 to 120 depending on the dataset. These scale factors are stored into model.info.scale_factor."

I thought the scale_factor should be the std value of all the offset data in a class. However, checking the the std value of the `bird.full.npz` dataset and the scale_factor in [bird.gen.json](https://storage.googleapis.com/quickdraw-models/sketchRNN/models/bird.gen.json), I found they are different (41.006386 - 71.3754). 

How can I calculate the correct scale_factor?

**Comment by notwaldorf** [comment]
@hardmaru, could you please take a look? thanks!
